### DISCUSSION TITLE
How to Reset Admin UI Configurations When Using a Config File via IMMICH_CONFIG_FILE?

### DISCUSSION THREAD

**Discussion by happyTonakai** [immich-app/immich#15002] [Q&A] [answered]
Hello,

I was using Immich v1.100.0 with a custom immich-machine-learning service that uses the CLIP model VIT-B-16. After updating to the latest version v1.123.0, the server container fails to start with the log message "Unknown model name".

To solve this, I created a config file named `immich.json` with all default configurations and set the environment variable `IMMICH_CONFIG_FILE` to point to this file. This allowed the server to start. However, due to PR #3836, I am now unable to modify the settings in the admin UI.

The config file was intended as a temporary solution, and I would like to revert the admin UI's configuration to its default state. Is there a way to reset or override the admin UI configurations when using a config file via the IMMICH_CONFIG_FILE environment variable?

Additionally, I tried rolling back to v1.100.0 to change the configurations in the admin UI of the older version. However, the server failed to start, stating that it could not connect to the database, so I had to abandon this approach.

Any guidance or workarounds would be greatly appreciated!

**Comment by danieldietzler** [comment, marked as answer]
That is a good point you're bringing up there. Unfortunately, the only way to fix this currently is by manually updating the database. We have docs on how to get the current values (https://immich.app/docs/guides/database-queries#system-config). `value` will end up being a JSON object, so SQL `UPDATE`ing the `system-config` key with the updated JSON object should do the trick. If you need any further help with how to go about this, feel free to ask :)

**Reply by happyTonakai** [reply]
Thank you! It works!
```sql
SELECT "key", "value" FROM "system_metadata" WHERE "key" = 'system-config';

UPDATE "system_metadata"
SET "value" = jsonb_set("value"::jsonb, '{machineLearning,clip,modelName}', '"ViT-B-16__openai"')
WHERE "key" = 'system-config';
```

**Reply by danieldietzler** [reply]
Awesome, great job!
